zebedee-cms/src/test/java/com/github/onsdigital/zebedee/model/publishing/client/PublishingRequestBuilderImplTest.java

```diff
@@ -86,6 +86,6 @@ public void testCreateGetContentHashRequest_success() throws Exception {
         assertThat(getRequest.getURI().getHost(), equalTo("localhost"));
         assertThat(getRequest.getURI().getPath(), equalTo("/contentHash"));
         assertThat(getRequest.getURI().getQuery(), equalTo("transactionId=transactionId&uri=uri"));
-        assertThat(getRequest.getFirstHeader("trace_id"), is(IsNull.notNullValue()));
+        assertThat(getRequest.getFirstHeader("X-Request-Id"), is(IsNull.notNullValue()));
```

Comment:
Not really your fault, it's the framework, but I hate the way `is(IsNull.notNullValue())` reads. It ugly! 😬